src/engraving/rw/read400/tread.cpp

```diff
@@ -3132,7 +3132,17 @@ bool TRead::readProperties(Note* n, XmlReader& e, ReadContext& ctx)
     } else if (tag == "head") {
         TRead::readProperty(n, e, ctx, Pid::HEAD_GROUP);
     } else if (tag == "velocity") {
-        n->setUserVelocity(e.readInt());
+        if (n->score()->mscVersion() >= 400) {
```

Comment:
Shouldn't we add a similar code to read206.cpp?

Reply:
Yes, thanks - done.  Since that code won't be executed for MU4 scores, I don't bother checking version and just skip the element.